pages/common/views/app.jsx

```diff
@@ -8,16 +8,20 @@ const App = ({
   scripts,
   crumbs,
   url,
-  reducers
+  reducers,
+  noScripts
 }) => (
   <Provider store={ store }>
     <Fragment>
       <script dangerouslySetInnerHTML={{__html: `window.INITIAL_STATE=${JSON.stringify(store.getState())}; window.REDUCERS = ${JSON.stringify(reducers)}`}} />
       <Layout
-        scripts={[
-          '/public/index.js',
-          `${url}/assets/index.js`
-        ]}
+        scripts={!noScripts
+          ? [
+            '/public/index.js',
+            `${url}/assets/index.js`
+          ]
+          : undefined
+        }
```

Comment:
Rather than a `noScripts` prop, it might be better to be able to overwrite the default scripts.

So you'd pass `scripts={[]}` instead to include no scripts. This would allow pages to include their own custom scripts as well if needed.

Reply:
Actually, it might be better generally to have the scripts _always_ passed in as a prop, and not hardcoded into the template at all. 

```
res.locals.scripts = ['/public/index.js', `${res.locals.url}/assets/index.js`];
```